### DISCUSSION TITLE
"Missing Property" Using two Spline Components in React!

### DISCUSSION THREAD

**Discussion by hasuwini77** [splinetool/react-spline#207] [Q&A]
Hi guys,

I'm super happy to use spline for my school project and I'm loving it so far. I just have a question, is it possible to use more than one spline component on the frontend using React? Without getting  an error in the console? 

Right now I'm getting: "Property Missing" 
window.console.error | @ | app-index.js:33
-- | -- | --
  | console.error | @ | hydration-error-info.js:63
  | overrideMethod | @ | console.js:288
  | buildTimeline | @ | runtime.js:7348
  | eval | @ | runtime.js:7348
  | setTimeout |   |  
  | play

it in load |   |  
-- | -- | --
  | t | @ | react-spline.js:63
  | eval | @ | react-spline.js:67
  | commitHookEffectListMount | @ | react-dom.development.js:21102
  | invokePassiveEffectMountInDEV | @ | react-dom.development.js:23980
  | invokeEffe

and that's one of the Spline component I'm using: 

`"use client";
import styled from "styled-components";
import Spline from "@splinetool/react-spline";

const SplineContainer = styled.div`
  width: 400px;
  height: 400px;
  margin: 0;
  padding: 0;
  display: flex;
  justify-content: center;
`;
export default function SplineDonation() {
  return (
    <SplineContainer>
      <Spline scene="https://prod.spline.design/MHTinSg3Dv0MZjiC/scene.splinecode" />
    </SplineContainer>
  );
}
`

Just a note that I'm seeing both elements rendering on my frontend on the same page so no problem with that, I'm just annoyed when I see JS dev console errors so I always try to remove them...

Here is a screenshot : 

<img width="313" alt="Screenshot 2024-08-19 at 22 35 04" src="https://github.com/user-attachments/assets/0c36e3bc-e1ee-45d2-a89e-7c15a7dc11ae">

**Comment by Stianlars1** [comment]
I also have this issue!

**Comment by MuttakinHasib** [comment]
Facing the same issue. Any solution?
